src/Datadog.Trace/RuntimeMetrics/MetricsNames.cs

```diff
@@ -28,5 +28,14 @@ internal static class MetricsNames
         public const string CpuUserTime = "runtime.dotnet.cpu.user";
         public const string CpuSystemTime = "runtime.dotnet.cpu.system";
         public const string CpuPercentage = "runtime.dotnet.cpu.percent";
+
+        public const string CurrentRequests = "runtime.dotnet.aspnetcore.requests.current";
```

Comment:
nit: Can we prefix these variables with `AspNetCore`?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1074/commits/5ed0ba4effebc9ea07000a7024aab83f1e8e1f88